.github/workflows/updateCopyrightDate.yml

```diff
@@ -0,0 +1,35 @@
+name: Update Copyright Notice
+
+on:
+  schedule:
+    - cron: '0 0 1 1 *' # Runs at midnight on the first day of every year
+  workflow_dispatch:
+
+jobs:
+  update-copyright:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Update copyright notice
+      run: |
+        pwsh -Command "
+        $configFilePath = 'config.toml'
+        $configContent = Get-Content -Path $configFilePath
+        $currentYear = (Get-Date).Year
+        $PreviousYear = (Get-Date).Year - 1
+        $UpdatedContent = $Content -replace $PreviousYear, $NewYear
```

Comment:
This should probably make sure it changes the number on only the correct row so there isn't a value of say 2026 in some other place that should not change. 🤔